src/fparser/two/Fortran2003.py

```diff
@@ -4970,6 +4995,7 @@ def tofortran(self, tab='', isfix=None):
             else:
                 tmp.append(item.tofortran(tab=tab+'  ', isfix=isfix))
         tmp.append(end.tofortran(tab=tab, isfix=isfix))
+        py2_encode_list_items(tmp)
```

Comment:
Add a test for this as was done for r438 and r808?

Reply:
Yes. I've also moved the associated where-construct test out of the test_fortran2003.py file.